devops/openshift/forms-flow-ai/bpm-deploy.yml

```diff
@@ -3,6 +3,25 @@ kind: Template
 metadata:
   name: forms-flow-bpm
 objects:
+  - apiVersion: v1
+    data:
+      mail-config.properties: |
+        # send mails via SMTP
+        mail.transport.protocol=smtp
+        mail.smtp.host=${SMTP_SERVER}
+        mail.smtp.port=25
```

Comment:
Is it ok to use a non SSL protect SMTP server? Is it for DEV only?

Reply:
this is the common config for gov services. I will confirm if we have alternatives but this is commonly used

Reply:
I am not sure, but we used the same server in our previous mail config too